src/components/credentials/credentialListItem.js

```diff
@@ -127,19 +128,17 @@ class CredentialListItem extends React.Component {
       case 'sources':
         (item.sources || []).sort((item1, item2) => item1.name.localeCompare(item2.name));
         return (
-          <Grid fluid>
+          <Grid className="container-fluid">
```

Comment:
Yeah I think having the tooltip on the text makes more sense! The larger area around it would help a user toggle it more easily. @bclarhk what do you think of wrapping the tooltip around the string content instead of the icon?